openstack/octavia/values.yaml

```diff
@@ -27,6 +27,7 @@ db_name: octavia
 
 proxysql:
   mode: ""
+  forceSecretCreation: true
```

Comment:
This can be dropped as well, no?

Reply:
i think it is better to be kept for the future, if anything changed in the secret, the secret will get updated before the job gets triggered , or is there a reason  to trigger the job before updating the proxysql config ?

Reply:
the job has already a weight of -1 which force helm to change the normal upgrade order

Reply:
well in the case of renaming even  `"helm.sh/hook-delete-policy": before-hook-creation` cannot remove the resource 
so should i remove the hook and we live will the risk that the job will be triggered before the secret got updated ? or keep it and naming changes will not remove the old secret ?

Reply:
I'd remove the hook in the secret. If a job is relying on the secret to be there, then it fails right away and we can fix it accordingly.